src/app/rundown-view/components/mini-shelf/mini-shelf.component.scss

```diff
@@ -3,46 +3,58 @@
   max-width: min-content;
   padding: 0;
   margin: 5px 0 5px 20px;
-  border-radius: 5px;
   overflow: hidden;
-  border: solid 1px var(--gray-color);
   background-color: var(--blue-color);
-  box-shadow: 0 2px 2px 2px var(--gradient-black-rgba-with-02-opacity-color);
+  border-radius: 5px;
+  border: solid 2px var(--gray-color);
+  box-shadow: 0 0 2px 2px var(--gradient-black-rgba-with-02-opacity-color);
 
   &:hover {
     cursor: pointer;
-    border: solid 1px var(--white-color);
-    box-shadow: 0 0 3px 3px var(--gradient-green-rgba-with-04-opacity-color);
+    box-shadow: 0 0 2px 2px var(--gradient-green-rgba-with-04-opacity-color);
+  }
+
+  &.is-on-air {
```

Comment:
So far, in the scss files, we haven't prefixed things like these with `is`, so maybe these should just be renamed like so:
`is-on-air` => `on-air`
`is-next` => `next`